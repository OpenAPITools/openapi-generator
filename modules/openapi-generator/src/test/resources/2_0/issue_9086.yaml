swagger: '2.0'
info:
  title: 'Buggy Api'
  version: '1.0'
consumes:
  - application/json
paths:
  /foo/bar:
    post:
      responses:
        '200':
          description: ok
          schema:
            type: object
            additionalProperties: false
            properties:
              n:
                type: number
                example: 1.23
  /foo/bar2:
    post:
      responses:
        '200':
          description: ok
          schema:
            $ref: '#/definitions/bar2'
definitions:
  bar2:
    type: object
    additionalProperties: false
    properties:
      n:
        type: number
        example: 4.56
