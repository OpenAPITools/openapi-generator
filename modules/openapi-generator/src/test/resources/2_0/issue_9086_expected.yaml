openapi: 3.0.1
info:
  title: 'Buggy Api'
  version: '1.0'
paths:
  /foo/bar:
    post:
      responses:
        '200':
          description: ok
          content:
            '*/*':
              schema:
                $ref: '#/components/schemas/inline_response_200'

  /foo/bar2:
    post:
      responses:
        '200':
          description: ok
          content:
            '*/*':
              schema:
                $ref: '#/components/schemas/bar2'
components:
  schemas:
    bar2:
      type: object
      example: { n: 4.56 }
      properties:
        n:
          type: number
          example: 4.56
    inline_response_200:
      type: object
      example: { n: 1.23 }
      properties:
        n:
          type: number
          example: 1.23
